etna/images/migrations/0002_migrate_old_image_data_to_new_model.py

```diff
@@ -80,8 +80,9 @@ class Migration(migrations.Migration):
         # Copy data from existing tables
         migrations.RunPython(migrate_forwards, migrations.RunPython.noop),
         # When running forwards, increment autoid to reflect additions
-        migrations.RunSQL(
-            'SELECT setval(pg_get_serial_sequence(\'"images_customimage"\',\'id\'), coalesce(max("id"), 1), max("id") IS NOT null) FROM "images_customimage";',
-            migrations.RunSQL.noop,
-        ),
+        # TODO: Figure out why this breaks...
+        # migrations.RunSQL(
+        #     'SELECT setval(pg_get_serial_sequence(\'"images_customimage"\',\'id\'), coalesce(max("id"), 1), max("id") IS NOT null) FROM "images_customimage";',
+        #     migrations.RunSQL.noop,
+        # ),
```

Comment:
Where is this failing for you? I've uncommented it but I'm not getting any errors from a fresh build... Maybe another Mac/Windows inconsistency?

Reply:
Could be, but this migration is trying to select data from a table that is empty to start with. On my machine, it always failed.

I'm not sure I really understand what this part of the migration is trying to do. Seems like a hacky way to fix existing data (if there is some) which is not _really_ what these type of migrations are for.